From 8a552d6acf08e708c4545d943d205dde00281fe2 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Sat, 17 Jan 2015 16:41:30 -0800
Subject: [PATCH 3/5] MacPorts Only: Don't embed the deployment target in the
 asm when using -fno-integrated-as

http://llvm.org/bugs/show_bug.cgi?id=21636

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
(cherry picked from commit ff5040cbee319c918097b40312ca3c9d6ba5f19a)
---
 lib/CodeGen/AsmPrinter/AsmPrinter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git llvm_release_35/lib/CodeGen/AsmPrinter/AsmPrinter.cpp macports_release_35/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
index 424e759..f766513 100644
--- llvm_release_35/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
+++ macports_release_35/lib/CodeGen/AsmPrinter/AsmPrinter.cpp
@@ -186,7 +186,7 @@ bool AsmPrinter::doInitialization(Module &M) {
   // use the directive, where it would need the same conditionalization
   // anyway.
   Triple TT(getTargetTriple());
-  if (TT.isOSDarwin()) {
+  if (TT.isOSDarwin() && MAI->useIntegratedAssembler()) {
     unsigned Major, Minor, Update;
     TT.getOSVersion(Major, Minor, Update);
     // If there is a version specified, Major will be non-zero.
-- 
2.2.2

